src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/transfers/TransferResponse.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.model.prison.transfers
+
+data class TransferResponse(
+  val prisonNumber: String,
+  val location: String?
+)
```

Comment:
needs swagger doc?

Reply:
for this I will create separate PR as we found out transfers do not return anything now and I plan to add this in next PR to not block frontend work

Reply:
done